internal/worker/remoterelationconsumer/remoteunitrelations/worker.go

```diff
@@ -5,34 +5,55 @@ package remoteunitrelations
 
 import (
 	"context"
+	"time"
 
 	"github.com/juju/clock"
+	"github.com/juju/collections/transform"
 	"github.com/juju/errors"
-	"github.com/juju/names/v6"
 	"github.com/juju/worker/v4"
 	"github.com/juju/worker/v4/catacomb"
 	"gopkg.in/macaroon.v2"
 
 	"github.com/juju/juju/api/watcher"
+	coreapplication "github.com/juju/juju/core/application"
+	corelife "github.com/juju/juju/core/life"
 	"github.com/juju/juju/core/logger"
+	corerelation "github.com/juju/juju/core/relation"
 	"github.com/juju/juju/rpc/params"
 )
 
 // RelationUnitChange encapsulates a remote relation event,
 // adding the tag of the relation which changed.
 type RelationUnitChange struct {
```

Comment:
Is this the same as the domain type with the same name? Might as well use that one.

Reply:
Not quite.